scripts/pkg/update-docs.go

```diff
@@ -40,6 +40,12 @@ var (
 	DryRun      bool
 )
 
+func getFilePrefixes() []string {
+	// All Relative links present in .md files inside content/docs/<version>/ to following
+	//directories and files will be translated into their appropriate absolute links.
+	return []string{"build/", "pkg/", "hack/", "ci/", "CHANGELOG", "LICENSE"}
+}
```

Comment:
probably better to stick to a variable in the `fixupMarkdownFile` function:
```go
filePrefixes := []string{"build/", "pkg/", "hack/", "ci/", "CHANGELOG", "LICENSE"}
```

otherwise the function name (`getFilePrefixes`) doesn't provide enough context

Reply:
done.